programs/ssl/ssl_client2.c

```diff
@@ -217,10 +216,19 @@ int main( void )
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
 
 #if defined(MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED)
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) 
+#define USAGE_PSK_RAW                                               \
+    "    psk=%%s                  default: \"\" (disabled)\n"     \
+    "                             The PSK values are in hex, without 0x.\n" \
+    "    psk_identity=%%s         default: \"Client_identity\"\n" \
+    "    key_exchange_modes=%%s   default: all\n"     \
+    "                             options: psk, psk_dhe, ecdhe_ecdsa, psk_all, all\n" 
+#else 
 #define USAGE_PSK_RAW                                               \
     "    psk=%%s              default: \"\" (disabled)\n"     \
     "                          The PSK values are in hex, without 0x.\n" \
-    "    psk_identity=%%s     default: \"Client_identity\"\n"
+    "    psk_identity=%%s     default: \"Client_identity\"\n" 
+#endif /* MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL */
```

Comment:
The existing documentation for `psk` and `psk_identity` can be retained - we only need to add the documentation of `key_exchange_modes` conditional on `MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL`. 
```c
#define USAGE_PSK_RAW                                               \
    "    psk=%%s                  default: \"\" (disabled)\n"       \
    "                             The PSK values are in hex, without 0x.\n" \
    "    psk_identity=%%s         default: \"Client_identity\"\n"
#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
#define USAGE_KEX_MODES \
    "    key_exchange_modes=%%s   default: all\n"     \
    "                             options: psk, psk_dhe, ecdhe_ecdsa, psk_all, all\n" 
#else 
#define USAGE_KEX_MODES 
#endif /* MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
```
and then `USAGE_KEX_MODES` needs to be added somewhere below.

Reply:
This would go well next to `USAGE_EARLY_DATA` and friends below.